robot-service/src/main/java/services/controllers/SelectCategories.java

```diff
@@ -0,0 +1,25 @@
+package services.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import services.converter.BookConverter;
+import services.model.response.BooksResponse;
+
+import java.util.List;
+
+@RestController
+public class SelectCategories {
+
+   @Autowired
+    private BookConverter bookConverter;
+
+    @RequestMapping(method = RequestMethod.GET, value = "categories")
+    public BooksResponse setCategories(@RequestParam(value = "categories")List<String> categories,
```

Comment:
It's because our concept. We want to filter our books by catagories. But now we cannot do this because we don't have api from db yet. It will be new feature in next days :facepunch: